client/packages/android/app/src/main/java/org/openmsupply/client/RemoteServer.java

```diff
@@ -12,6 +14,7 @@ public RemoteServer() {
     }
 
     public void start(int port, String filesDir, String cacheDir, String androidId) {
+        Logger.info("Starting OMS Rust Server");
```

Comment:
At one stage It seemed like OMS wasn't starting, would be nice to get some kind of feedback in `logcat` about OMS server status. Maybe an R&D or backlog issue?